src/js/views/admin/shared/clinicians_components.js

```diff
@@ -63,6 +66,107 @@ const RoleComponent = Droplist.extend({
   },
 });
 
+const GroupsItemView = View.extend({
+  tagName: 'li',
+  className: 'clinician-groups__item',
+  template: hbs`<span class="clinician-groups__item-icon">{{far "users"}}</span>{{name}}<button class="button--icon clinician-groups__remove js-remove">{{far "trash-alt"}}</button>`,
+  triggers: {
+    'click .js-remove': 'remove',
+  },
+});
+
+const ClinicianGroupsView = CollectionView.extend({
+  tagName: 'ul',
+  childView: GroupsItemView,
+  childViewTriggers: {
+    'remove': 'remove',
+  },
+  onRemove(child) {
+    this.triggerMethod('remove:clinician', child.model);
+  },
+});
+
+const GroupsDropList = Droplist.extend({
+  initialize() {
+    this.listenTo(this.collection, 'update', this.show);
+  },
+  viewOptions: {
+    className: 'button-secondary clinician-groups__droplist',
+    template: hbs`{{far "users"}}Add Group...`,
+  },
+  picklistOptions: {
+    attr: 'name',
+  },
+  onShow() {
+    if (this.getOption('clinician').isNew()) {
+      this.setState({ isDisabled: true });
+      return;
+    }
+
+    this.setState({ isDisabled: this.collection.length === 0 });
+  },
+  onPicklistSelect({ model }) {
+    this.triggerMethod('add:clinician', model);
+    this.popRegion.empty();
+  },
+});
+
+const GroupsComponent = View.extend({
+  template: hbs`
+    <div data-groups-region></div>
+    <div data-droplist-region></div>
+  `,
+  regions: {
+    groups: '[data-groups-region]',
+    droplist: '[data-droplist-region]',
+  },
+  initialize() {
+    this.currentClinician = Radio.request('bootstrap', 'currentUser');
+    this.clinician = this.getOption('clinician');
+    this.clinicianGroups = this.clinician.getGroups();
+    this.groups = this.currentClinician.getGroups();
+
+    this.groups.remove(this.clinicianGroups.models);
+
+    this.listenTo(this.clinicianGroups, {
+      'add': this.onAddClinicianGroup,
+      'remove': this.onRemoveClinicianGroup,
+    });
+  },
+  onRender() {
+    this.showGroups();
+    this.showDroplist();
+  },
+  showGroups() {
+    const groupsView = this.showChildView('groups', new ClinicianGroupsView({
+      collection: this.clinicianGroups,
+    }));
+
+    this.listenTo(groupsView, 'remove:clinician', this.removeClinicianGroup);
+  },
+  showDroplist() {
+    const droplist = this.showChildView('droplist', new GroupsDropList({
+      collection: this.groups,
+      clinicianGroups: this.clinicianGroups,
+      state: this.getOption('state') || {},
```

Comment:
if we pass the state down this should likely be `state: this.getState().attributes,`